examples/all.py

```diff
@@ -166,6 +166,7 @@ def run_example(example, *, reporter=None):
         mod = load_example_module(example)
         if reporter:
             suppress_output(mod.main)
+
```

Comment:
I don't think this empty line is needed.

Reply:
deleted that line to resolve that issue